### PR TITLE
Adds information for command history and airgap version requirements.

### LINKED ISSUE #804: Add supported versions info to `mc support` doc
Add the following info from @anjalshireesh to the docs, probably at the `mc support` overview page or possibly on the deprecated commands page.

## Health report

RELEASE.2020-10-03T02-54-56Z -> `mc admin health`
RELEASE.2020-11-17T00-39-14Z -> `mc admin subnet health`
RELEASE.2022-02-13T23-26-13Z -> `mc support diag`

## Online Registration

RELEASE.2021-11-05T10-05-06Z -> `mc admin subnet register`
RELEASE.2022-02-13T23-26-13Z -> `mc support register`
RELEASE.2022-07-15T09-20-55Z -> `mc license register`

## Airgapped Registration

The current airgapped registration flow in SUBNET will work with `mc` version RELEASE.2022-07-29T19-17-16Z onwards. Older versions had a different flow that doesn't work anymore.

### PR DESCRIPTION
**Pull request by djwfyi**
- Updates troubleshooting doc for airgap registration
- Updates `mc license register` doc with airgap registration and command history
- Updates `mc support diag` with command history

Closes #804

### CODE REVIEW

File: source/reference/minio-mc/mc-license-register.rst [hunk 1]
```diff
@@ -56,6 +82,11 @@ Register a Deployment Without Direct Internet Access
 
 Register a MinIO deployment at alias ``minio3`` on SUBNET that does not have direct Internet access due to a firewall, airgap, or the like.
 
+.. versionchanged:: mc RELEASE.2022-07-29T19-17-16Z
+
+   The current airgap registration process works with MinIO Client version ``RELEASE.2022-07-29T19-17-16Z`` or later.
```
Comment (feorlen) [inline source/reference/minio-mc/mc-license-register.rst:87]: Perhaps drop `current` here? How likely are we to make older versions work in the future?

**Review by feorlen — APPROVED** [review]
Looks good! One potential tweak in wording you might like.

File: source/reference/minio-mc/mc-support-diag.rst [hunk 2]
```diff
@@ -23,7 +50,7 @@ The :mc-cmd:`mc support diag` command generates a health report for a MinIO depl
 
 .. end-mc-support-diag-desc
 
-For deployments registered with the MinIO subscription network (|subnet-short|), the command can automatically upload the health report for analysis.
+For deployments registered with the MinIO subscription network (|subnet-short|), the command can automatically upload the health report for analysis if you enable :mc-cmd:`~mc support callhome`.
```
Comment (anjalshireesh) [inline source/reference/minio-mc/mc-support-diag.rst:53]: The command always uploads to SUBNET irrespective of callhome enabled or not.
If callhome is enabled, diagnostics data is uploaded automatically every 24hrs (without running any command).

**Review by anjalshireesh — CHANGES_REQUESTED** [review]

File: source/reference/minio-mc/mc-license-register.rst (hunk 1 above)
Comment (djwfyi) [reply source/reference/minio-mc/mc-license-register.rst:87]: Agree on dropping it. Our docs always target current release anyway.

File: source/operations/troubleshooting.rst [hunk 3]
```diff
@@ -79,6 +82,9 @@ Registering Your MinIO Deployment with SUBNET
 
       For clusters without direct Internet access, refer to the instructions in the :ref:`airgap example <minio-license-register-airgap>` of the :mc:`mc license register` documentation.
 
+      The process to register a deployment for SUBNET with an airgap works with MinIO Client version ``RELEASE.2022-07-29T19-17-16Z`` or later.
```
[the remark this answers is not in the document]
Comment (djwfyi) [reply source/operations/troubleshooting.rst:85]: Good idea. Done.

**Review by ravindk89 — APPROVED** [review]

**Review by anjalshireesh — APPROVED** [review]
LGTM

**Pull request merged**